Makefile

```diff
@@ -74,6 +76,16 @@ vendor:
 $(GOPATH)/bin/binappend-cli:
 	GO111MODULE=off go get -u github.com/yourfin/binappend-cli
 
+# Get JQ based on the platform (linux or darwin)
+$(JQ):
+ifeq ($(GOOS),linux)
+	@curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o jq
+endif
+ifeq ($(GOOS),darwin)
+	@curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-osx-amd64 -o jq
+endif
+	@chmod +x jq
```

Comment:
Instead adding the `jq` dependency can we just add the place holder in the sample file and use `envsubst` ?

```bash
$ cat release.json 
{
	"description": "Bringing a minimal OpenShift 4.1 or newer cluster to your local laptop or desktop computer.",
	"version": {
		"crcVersion": "$crcVersion",
		"gitSha": "$gitSha"
	},
	"openshiftVersion": "$openshiftVersion"
}

$ crcVersion=1.0.0-beta.5 gitSha=f2aa58c openshiftVersion=4.1.14 envsubst < release.json 
{
	"description": "Bringing a minimal OpenShift 4.1 or newer cluster to your local laptop or desktop computer.",
	"version": {
		"crcVersion": "1.0.0-beta.5",
		"gitSha": "f2aa58c"
	},
	"openshiftVersion": "4.1.14"
}
```

Reply:
Or the more usual `sed`?

Reply:
besides, using a 'released' version like this needs trust. I'd rather go for a packaged version... but as I said before, not at all. rather not see another dependency

Reply:
Went with using `sed`, as with that we can avoid having to pass the envs.